faker/providers/person/en_IN/__init__.py

```diff
@@ -2,225 +2,554 @@
 
 
 class Provider(PersonProvider):
-    formats = ("{{first_name}} {{last_name}}",)
 
-    # First names are from
-    #   https://www.babycenter.in/a25010193/modern-indian-baby-names
-    #   https://en.wikipedia.org/wiki/Category:Male_actors_in_Malayalam_cinema (not used exhaustively)
-    #   https://en.wikipedia.org/wiki/List_of_Tamil_film_actors (not used exhaustively)
-    # Last names are from https://www.familyeducation.com/baby-names/browse-origin/surname/indian
+    formats_male = ("{{first_name_male}} {{last_name}}",)
+    formats_female = ("{{first_name_female}} {{last_name}}",)
+    formats = ("{{first_name}} {{last_name}}",)
 
-    first_names = (
+    # https://www.in.pampers.com/pregnancy/baby-names/article/indian-baby-boys-names
```

Comment:
@fcurella source is added before each list. Do you want me to add all at the top?

Reply:
Sorry, I missed it. I think this is fine.